### ISSUE TITLE
Tensorflow code does not give the proper results

### ISSUE THREAD

**Issue by devraj89** [jiangqy/DCMH-CVPR2017#2]
Hi

I am trying to run the code for **bit = 64** you provided in the tensor flow environment. The **loss** is **decreasing** but the **accuracy (MAP)** is remaining the **same**. As you can observe below the loss has decreased quite significantly but the accuracy is renaming the same.

Can you please tell me if I am going wrong somewhere ?

```
...loading and splitting data finish
...epoch:   0, loss: 177061069.097
...training procedure starts
...epoch:   1, loss: 175336058.349, comment: update B
...training procedure finish
...test map: map(i->t): 0.543, map(t->i): 0.543
...epoch:   2, loss: 173065982.115, comment: update B
...training procedure finish
...test map: map(i->t): 0.543, map(t->i): 0.543
...epoch:   3, loss: 170820578.049, comment: update B
...training procedure finish
...test map: map(i->t): 0.543, map(t->i): 0.543
...epoch:   4, loss: 168648051.304, comment: update B
...training procedure finish
...test map: map(i->t): 0.543, map(t->i): 0.543
...
...epoch: 496, loss: 70586304.761, comment: update B
...training procedure finish
...test map: map(i->t): 0.543, map(t->i): 0.543
...epoch: 497, loss: 70586286.625, comment: update B
...training procedure finish
...test map: map(i->t): 0.543, map(t->i): 0.543
...epoch: 498, loss: 70586388.189, comment: update B
...training procedure finish
...test map: map(i->t): 0.543, map(t->i): 0.543
...epoch: 499, loss: 70586918.393, comment: update B
...training procedure finish
...test map: map(i->t): 0.543, map(t->i): 0.543
...epoch: 500, loss: 70586985.325, comment: update B
...training procedure finish
...test map: map(i->t): 0.543, map(t->i): 0.543
```

In addition I had to make the following changes to run your code in Tensor-flow : 
[1] changed **xrange** to **range** as I am using python 3
[2] put **print** statements inside quotes
[3] changed statements such as this 
`for iter in xrange(num_train / batch_size):`
to
`for iter in range(np.int(np.floor(num_train / batch_size))):`
to make the data type int
[4] defined the variable
`L['retrieval'] = labels[QUERY_SIZE:DATABASE_SIZE + QUERY_SIZE,:] `
